### ISSUE TITLE
pauseWhenHidden not working

### ISSUE THREAD

**Issue by ghost** [phoboslab/jsmpeg#402]
No matter if i put pauseWhenHidden true or false the audio stops nevertheless when i switch tabs

**Comment by shnarobe** [comment]
Yes, I experienced the same issue. According to the documentation: "**_pauseWhenHidden – whether to pause playback when the tab is inactive. Default true. Note that browsers usually throttle JS in inactive tabs anyway_**.",  but it doesn't work. The audio/video stops regardless of the value of 'pauseWhenHidden'.

**Comment by phoboslab** [comment]
> The audio/video stops regardless of the value of 'pauseWhenHidden'.

Yes, because

> browsers usually throttle JS in inactive tabs anyway

Afaik there's the nothing we can do about this.

**Comment by shnarobe** [comment]
Thank you very much for responding!
